.github/workflows/build.yml

```diff
@@ -98,7 +98,7 @@ jobs:
           tar -zcvf ${BUILD_FILE_NAME}.tar.gz ./${BUILD_FILE_NAME}
           mkdir -p output/artifacts
           cp ${BUILD_FILE_NAME}.tar.gz output/artifacts
-          sha256sum ${BUILD_FILE_NAME}.tar.gz | head -c 64 > output/artifacts/${BUILD_FILE_NAME}.sha256
+          sha256sum ${BUILD_FILE_NAME}.tar.gz | head -c 64 > output/artifacts/${BUILD_FILE_NAME}.tar.gz.sha256
           gpg --default-key ${GPG_KEY_ID} --sign --armor --output output/artifacts/${BUILD_FILE_NAME}.tar.gz.asc --detach-sig ${BUILD_FILE_NAME}.tar.gz
```

Comment:
Can we update this to follow a similar pattern with the windows route of having a `GZIP_FILE_NAME` instead of using `BUIILD_FILE_NAME` and repeating `.tar.gz` multiple times?

Reply:
Good idea, let me add thiat to this PR